pkg/gator/test/test_test.go

```diff
@@ -56,6 +56,20 @@ func init() {
 	}
 }
 
+func ignoreGatorResultFields() cmp.Option {
```

Comment:
the main difference here between our custom option and the `IgnoreFields` option is that the latter relies the existence of a field; ref: https://github.com/google/go-cmp/blob/a97318bf6562f2ed2632c5f985db51b1bc5bdcd0/cmp/cmpopts/struct_filter.go#L177-L179